Newtonsoft.json build warnings

```
18>C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3277: Found conflicts between different versions of "Newtonsoft.Json" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed.

```

the nuget package reference was updated to v12 but the csproj references are still using v9:
```
    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
      <HintPath>$(PackagesPath)\Newtonsoft.Json\lib\net45\Newtonsoft.Json.dll</HintPath>
      <Private>True</Private>
      <IncludeInVSIX>True</IncludeInVSIX>
    </Reference>
```

--- a/Build/16.0/packages.config
+++ b/Build/16.0/packages.config
@@ -4,7 +4,7 @@
     <package id="CommandLineParser" version="2.2.1" />
     <package id="Python2" version="2.7.15" />
     <package id="Python" version="3.6.6" />
-    <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net45" /> 
+    <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" /> 
     <!--These 2 packages are obtained from private sources. See nuget.config for more info -->
     <package id="Microsoft.VisualStudio.Python.LanguageServer" version="0.1.29" />
     <package id="Microsoft.Internal.VisualStudio.Shell.Embeddable" version="16.3.28923.271" />

